pkgs/data/fonts/lilex/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "lilex";
+  version = "2.200";
+
+  src = fetchzip {
+    url = "https://github.com/mishamyrt/Lilex/releases/download/${version}/Lilex.zip";
+    sha256 = "sha256-MPQfqCMFMjcAlMos1o4bC+I+cvQYwr2TjI4Q03QeuaQ=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.